src/data/alphabets/alphabets-items.js

```diff
@@ -0,0 +1,4 @@
+export default function() {
+  let i;
+  return [...Array(26)].map((_) => (++i).toString(36), (i = 9));
```

Comment:
you can just use [...'abcdefgh'] for it to be more readable and concise. Also try to refactor the name of this file to represent what is being returned, i.e. something like 'alphabets-array.js'.